### PR TITLE
🌱 Fix typo: kuberentes->kubernetes

### PR DESCRIPTION
**Pull request by bboreham**

### CODE REVIEW

File: bootstrap/kubeadm/config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml [hunk 1]
```diff
@@ -282,7 +282,7 @@ spec:
                     type: string
                   kubernetesVersion:
                     description: 'KubernetesVersion is the target version of the control
-                      plane. NB: This value defaults to the Machine object spec.kuberentesVersion'
+                      plane. NB: This value defaults to the Machine object spec.kubernetesVersion'
```
Comment (wfernandes) [inline bootstrap/kubeadm/config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml:285]: I know this just got merged. But is referring to https://github.com/kubernetes-sigs/cluster-api/blob/303f4923517c42a0fad9d40935eafcc151e0f288/api/v1alpha3/machine_types.go#L58-L61

Or is there another field called `spec.kubernetesVersion`?

@detiber @vincepri

Comment (bboreham) [reply bootstrap/kubeadm/config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml:285]: I can't find a field called `kubernetesVersion` anywhere back to v1alpha1.
There used to be separate kubelet and controlPlane versions: https://github.com/kubernetes-sigs/cluster-api/blob/f31486484d5b33c785540eeaffd47fe57832aef5/pkg/apis/cluster/v1alpha1/machine_types.go#L154-L163

Comment (wfernandes) [reply bootstrap/kubeadm/config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml:285]: Hmm...yeah that's why I asked about it. Should this be updated to `This value defaults to the Machine object spec.Version` unless I'm misunderstanding something?

Comment (vincepri) [reply bootstrap/kubeadm/config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml:285]: Let's file an issue, seems worth to investigate + fix.

Comment (wfernandes) [reply bootstrap/kubeadm/config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml:285]: Ok...I'll do that. Thanks!

**Pull request merged**